lib/chef/knife/uploader_base.rb

```diff
@@ -103,16 +103,21 @@ def self.included(includer)
         end
 
         option :pattern,
-          :short => '-p PATTERN',
-          :long => '--pattern PATTERN',
-          :description => 'A regular expression pattern to restrict the set of objects to ' +
+          short: '-p PATTERN',
+          long: '--pattern PATTERN',
+          description: 'A regular expression pattern to restrict the set of objects to ' +
                           'manipulate',
-          :proc => Proc.new { |value| Chef::Config[:knife][:pattern] = value }
+          proc: Proc.new { |value| Chef::Config[:knife][:pattern] = value }
 
         option :debug,
-          :long => '--debug',
-          :description => 'Turn on debug messages',
-          :proc => Proc.new { |value| Chef::Config[:knife][:debug] = value }
+          long: '--debug',
+          description: 'Turn on debug messages',
+          proc: Proc.new { |value| Chef::Config[:knife][:debug] = value }
+
+        option :show_encrypted,
+          long: '--show_encrypted',
+          description: 'Show the decrypted data bag contets when different.',
```

Comment:
The intent is whether or not to show the (decrypted) contents of encrypted data bags.  We always show the contents of  data bags that are not encrypted.  I think '--show_decrypted' might be interpreted to mean 'show data bag contents that are not encrypted' rather than 'show the decrypted contents of encrypted data bags'.  Although the help message spells that out pretty clearly...